packages/iov-bcp/src/transactions.ts

```diff
@@ -46,29 +46,29 @@ export function publicKeyBundleEquals(left: PublicKeyBundle, right: PublicKeyBun
 export type ChainId = string & As<"chain-id">;
 
 /** a public key we can identify with on a blockchain */
-export interface PublicIdentity {
+export interface Identity {
   readonly chainId: ChainId;
   readonly pubkey: PublicKeyBundle;
 }
 
-export function isPublicIdentity(data: any): data is PublicIdentity {
+export function isIdentity(data: any): data is Identity {
   return (
     typeof data === "object" &&
     data !== null &&
-    typeof (data as PublicIdentity).chainId === "string" &&
-    isPublicKeyBundle((data as PublicIdentity).pubkey)
+    typeof (data as Identity).chainId === "string" &&
+    isPublicKeyBundle((data as Identity).pubkey)
   );
 }
 
 /**
- * Compares two objects that conform to the PublicIdentity interface for equality.
- * This can also be used to compare pairs of derived types like LocalIdentity/PublicIdentity
- * or LocalIdentity/LocalIdentity in which case all non-PublicIdentity fields are ignored.
+ * Compares two objects that conform to the Identity interface for equality.
+ * This can also be used to compare pairs of derived types like LocalIdentity/Identity
+ * or LocalIdentity/LocalIdentity in which case all non-Identity fields are ignored.
```

Comment:
The last two lines are outdated. Could you generalize to someting like _non-Identity properies are ignored_, which is an important property I guess as it works differently than Jasmine's equality matcher, which can detect extra fields.